onedal/datatypes/_data_conversion.py

```diff
@@ -18,28 +18,43 @@
 
 import numpy as np
 
-from daal4py.sklearn._utils import make2d
 from onedal import _backend, _is_dpc_backend
 
-from ..utils import _is_csr
-
 
 def _apply_and_pass(func, *args, **kwargs):
     if len(args) == 1:
         return func(args[0], **kwargs)
     return tuple(map(lambda arg: func(arg, **kwargs), args))
 
 
-if _is_dpc_backend:
+def convert_one_to_table(arg):
+    return _backend.to_table(np.atleast_2d(arg) if np.isscalar(arg) else arg)
```

Comment:
It worth to add a comment. Will this properly work with dpnp ndarrays and dpctl usm_ndarrays?